node/src/components/small_network.rs

```diff
@@ -453,6 +453,8 @@ where
                     .open_connections
                     .set(self.incoming.len() as i64);
 
+                self.net_metrics.peers.set(self.peers().len() as i64);
```

Comment:
I think this might be best moved to [`handle_message()`](https://github.com/CasperLabs/casper-node/blob/23da6ee23c1f855dbe249dc0448e0a7effbb2661/node/src/components/small_network.rs#L682-L701) where we decide whether to keep a new connection or not based on the handshake message.  We'd probably want it called for the case where we remove a peer and where we keep a peer.

Also, I think we should add this metric to the new `network` component in this PR so it doesn't get forgotten about.  That should go in where we handle `Event::ConnectionEstablished` and `Event::ConnectionClosed` I suppose.

Reply:
It would appear that `NetworkingMetrics` are not yet implemented for the new `network` component. I think @marc-casperlabs mentioned that there should be a todo for that?